skin/src/main/java/org/researchstack/skin/ui/fragment/ActivitiesFragment.java

```diff
@@ -71,14 +79,27 @@ private void unsubscribe() {
     }
 
     private void setUpAdapter() {
-        unsubscribe();
 
         recyclerView.setLayoutManager(new LinearLayoutManager(recyclerView.getContext()));
         recyclerView.addItemDecoration(new DividerItemDecoration(recyclerView.getContext(),
                 DividerItemDecoration.VERTICAL_LIST,
                 0,
                 false));
 
+        fetchData();
+
+    }
+
+    /**
+     * Override this method to provide a customer adapter for your application.
+     * @return  The adapter for displaying the list of tasks.
+     */
+    protected TaskAdapter createTaskAdapter() {
```

Comment:
Nice, thanks.  That should be sufficient for someone who want to implement there own UI.